hexrdgui/image_series_toolbar.py

```diff
@@ -161,6 +177,17 @@ def update_omega_label_text(self):
 
         self.omega_label.setText(omega_label_text(*ome_range))
 
+    def update_back_forward_buttons(self, val):
+        self.back_button.setEnabled(self.slider.minimum() != val)
+        self.forward_button.setEnabled(self.slider.maximum() != val)
+
+    def change_frame(self, value):
```

Comment:
```suggestion
    def shift_frame(self, value):
```
Can you rename this so it's clear that we are shifting by the value, rather than changing *to* the value?